src/mixins/raster-layer.js

```diff
@@ -224,9 +224,9 @@ export default function rasterLayer(layerType) {
     // data structure, but probably not an issue given the amount
     // of popup col attrs to iterate through is small
     const dim = _layer.group() || _layer.dimension()
-    if (dim || _layer.layerType() === "points") {
+    if (dim || (_layer.layerType() === "points" || _layer.layerType() === "lines")) {
```

Comment:
This common-type check would be good to abstract, if it's done in more places.

Reply:
the utils functions were in immerse not in charting. I will add the similar functions if we have more of this checks.